src/test/java/com/wikia/webdriver/testcases/discussions/CreatingReplyTests.java

```diff
@@ -74,11 +74,11 @@ public void userOnDesktopCanCreateReplyOnPostDetailsPage() {
   // Testing methods
 
   private void userOnDesktopMustBeLoggedInToUseReplyCreator(final ReplyCreator replyCreator) {
-    anonymousUserOnReplyEditorClickIsRedirectedTo(replyCreator, MercurySubpages.REGISTER_PAGE);
+    anonymousUserOnReplyEditorClickIsRedirectedTo(replyCreator, MobileSubpages.REGISTER_PAGE);
   }
 
   private void userOnMobileMustBeLoggedInToUseReplyCreator(final ReplyCreator replyCreator) {
-    anonymousUserOnReplyEditorClickIsRedirectedTo(replyCreator, MercurySubpages.JOIN_PAGE);
+    anonymousUserOnReplyEditorClickIsRedirectedTo(replyCreator, MobileSubpages.JOIN_PAGE);
   }
 
   private void anonymousUserOnReplyEditorClickIsRedirectedTo(final ReplyCreator replyCreator, final String urlFragment) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "anonymousUserOnReplyEditorClickIsRedirectedTo" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

Reply:
if this method is unused please remove it

Reply:
it is used